src/commands/getdirs.ts

```diff
@@ -7,13 +7,24 @@ export const command = 'getdirs';
 export const describe = 'get paths for project-relevant directories';
 export const builder = {};
 
+/**
+ * view relevant directories (contracts, build, etc.)
```

Comment:
Is this the good doc? Same as lower but for "handler"?

Reply:
I followed the same pattern of just slightly modifying comments for handler, and the function that the handler calls. 

I'm not sure I understand your question here.